.circleci/config.yml

```diff
@@ -67,3 +67,6 @@ workflows:
           filters: *filter-only-source
     jobs:
       - deploy-website
+  check_build:
+    jobs:
+      - check-build
```

Comment:
It seems every commit in this repo will trigger the build, but PR from 3rd party will not trigger this workflow, is this correct?

Given Circle CI does not [support build](https://ideas.circleci.com/ideas/CCI-I-316) directly on PR, I agree with the current solution.

Reply:
I will take this as an approval

Reply:
@kaichaosun I think this is good enough for now, but you raise a good point.

https://discuss.circleci.com/t/how-to-trigger-pr-pipelines-in-circleci-for-bitbucket/31715 gave me the idea that we could use github's webhooks to trigger ci builds when PRs are open. Maybe that's worth exploring if the current solution becomes a bottleneck.